mappings/net/minecraft/client/resource/ClientResourcePackCreator.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS dme net/minecraft/client/resource/ClientResourcePackCreator
```

Comment:
Creator? Not Downloader?

Reply:
This also creates resource packs for the default pack and the old programmer art pack.